src/Commands/SetTempUrlKey.php

```diff
@@ -48,7 +49,13 @@ class SetTempUrlKey extends Command
     public function handle(): void
     {
         try {
-            $this->container = Storage::disk($this->option('disk'))->getAdapter()->getContainer();
+            $adapter = Storage::disk($this->option('disk'))->getAdapter();
+
+            if ($adapter instanceof CachedAdapter) {
+                $adapter = $adapter->getAdapter();
```

Comment:
There are two calls to `getAdapter`. Is this correct?

Reply:
Yes, same as https://github.com/illuminate/filesystem/blob/8eaa230711347ac76b3b61372d1b3b332e0931ee/FilesystemAdapter.php#L426

I'll make sure this works as intended when I write tests.

Reply:
Excellent! Thanks :+1: